apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/ConcertsList.test.tsx

```
@@ -0,0 +1,477 @@
+import React from 'react';
```

ESLint

[error] Filename is not in pascal case. Rename it to `ConcertsList.Test.tsx`.

(unicorn/filename-case)